mods/persistence/modules/mob/new_player.dm

```diff
@@ -118,6 +119,11 @@
 		qdel(src)
 		return
 
+
+	switch(alert("Are you sure you want to join the game with the character you've created? This cannot be undone!",,"Yes","No"))
```

Comment:
Still got an extra "," here before "Yes"

Reply:
alert(Usr=usr,Message,Title,Button1="Ok",Button2,Button3)  if I removed the coma, wouldn't it use "yes" as the title argument?

Reply:
Yes, so add a title for cleanliness I suppose. "Character Confirmation" or such would work.